frontend/src/pages/Register.jsx

```diff
@@ -1,5 +1,12 @@
 import { Link } from "react-router-dom";
+import RegisterForm from "../components/RegisterForm";
 
 export default function Register() {
-  return <Link to="/account">Compte utilisateur</Link>;
+  return (
+    <div className="bg-black text-white">
+      <img src="../../src/assets/img/logo-black.jpg" alt="logo" />
```

Comment:
You're going a little too high with this path

Reply:
unfortunately any other path doesn't work 😢

Reply:
use import and variable